ExposureNotification.App/ExposureNotification.App/ExposureNotification.App.csproj

```
@@ -15,6 +15,10 @@
 		<EmbeddedResource Include="Resources\Fonts\fa-brands-400.ttf" />
 		<EmbeddedResource Include="Resources\Fonts\fa-regular-400.ttf" />
 		<EmbeddedResource Include="Resources\Fonts\fa-solid-900.ttf" />
+		<EmbeddedResource Include="settings.json" Condition="'$(Configuration)' == 'Debug'" />
+		<EmbeddedResource Include="settings.json" Condition="'$(Configuration)' != 'Debug'">
+			<Link>local.settings.json</Link>
+		</EmbeddedResource>
```

```suggestion
		<EmbeddedResource Include="local.settings.json" Link="settings.json" Condition="'$(Configuration)' != 'Debug'" />
```

Maybe it is backwards?